q2cli/builtin/tools.py

```diff
@@ -7,13 +7,11 @@
 # ----------------------------------------------------------------------------
 
 import os
-
```

Comment:
Please put this blank line back in - it was being used to group std lib imports (like os) from 3rd-party imports (like click).